templates/config.edn

```diff
@@ -1,137 +1,171 @@
- ;; Default is true, you can also toggle this via setting page
+ ;; Set a custom date format for the journal page title.
+ ;; Example usage:
+ ;; :journal/page-title-format "EEE do, MMM yyyy"
+ ;; e.g., "Tue 19th, Jan 2038"
+ ;; Default value: "MMM do, yyyy"
+ ;; e.g., "Jan 19th, 2038"
+ :journal/page-title-format "MMM do, yyyy"
+
+ ;; Specify the journal filename format using a valid date format string.
+ ;; !Warning:
+ ;;   This configuration is not retroactive and affects only new journals.
+ ;;   To show old journal files in the app, manually rename the files in the
+ ;;   journal directory to match the new format.
+ ;; Default value: "yyyy_MM_dd"
+ :journal/file-name-format "yyyy_MM_dd"
+
+ ;; Enable tooltip preview on hover.
+ ;; Default value: true
  :ui/enable-tooltip? true
 
- ;; Show brackets around page references
- ;; :ui/show-brackets? true
+ ;; Display brackets [[]] around page references.
+ ;; Default value: true
+ :ui/show-brackets? true
 
- ;; Enable showing the body of blocks when referencing them.
+ ;; Display the complete block content when referencing.
+ ;; Default value: false
  :ui/show-full-blocks? false
 
- ;; Expand block references automatically when zoom-in
+ ;; Automatically expand block references when zooming in.
+ ;; Default value: true
  :ui/auto-expand-block-refs? true
 
- ;; Enable Block timestamp
+ ;; Enable Block timestamps.
+ ;; Default value: false
  :feature/enable-block-timestamps? false
 
- ;; Enable remove accents when searching.
- ;; After toggle this option, please remember to rebuild your search index by press (cmd+c cmd+s).
+ ;; Disable accent marks when searching.
+ ;; After changing this setting, rebuild the search index by pressing (^C ^S).
+ ;; Default value: true
  :feature/enable-search-remove-accents? true
 
- ;; Enable journals
- ;; :feature/enable-journals? true
+ ;; Enable journals.
+ ;; Default value: true
+ :feature/enable-journals? true
 
- ;; Enable flashcards
- ;; :feature/enable-flashcards? true
+ ;; Enable flashcards.
+ ;; Default value: true
+ :feature/enable-flashcards? true
 
- ;; Enable Whiteboards
- ;; :feature/enable-whiteboards? true
+ ;; Enable whiteboards.
+ ;; Default value: true
+ :feature/enable-whiteboards? true
 
- ;; Disable the built-in Scheduled tasks and deadlines query
- ;; :feature/disable-scheduled-and-deadline-query? true
+ ;; Disable the journal's built-in 'Scheduled tasks and deadlines' query.
+ ;; Default value: false
+ :feature/disable-scheduled-and-deadline-query? false
 
- ;; Specify the number of days in the future to display in the
- ;; scheduled tasks and deadlines query, with a default value of 7 which
- ;; displays tasks for the next 7 days.
+ ;; Specify the number of days displayed in the future for
+ ;; the 'scheduled tasks and deadlines' query.
  ;; Example usage:
- ;; Display all scheduled and deadline blocks for the next 14 days
+ ;; Display all scheduled tasks and deadlines for the next 14 days:
```

Comment:
We should revert wording here. The new wording is incorrect as this functionality isn't limited to tasks